deeppavlov/core/commands/train.py

```diff
@@ -125,6 +144,8 @@ def train_model_from_config(config_path: str) -> None:
 
     if callable(getattr(model, 'train_on_batch', None)):
         _train_batches(model, iterator, train_config, metrics_functions)
+    elif callable(getattr(model, 'fit_batches', None)):
+        _fit_batches(model, iterator, train_config)
     elif callable(getattr(model, 'fit', None)):
         _fit(model, iterator, train_config)
```

Comment:
All estimators now have the `fit_batches` so this case block will never reach the `fit` part, but most of those estimators still want to fit the old way and just `pass` in the `fit_batches` method so training would not happen.